### PR TITLE
map tools: data verification tool, first ten files

### PR DESCRIPTION
**Pull request by ooooooops**

### CODE REVIEW

**Comment by xiaoxq** [comment]
LGTM on coding.

File: modules/map/tools/map_datachecker/server/alignment.h [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    } else {
+      pose_info.end_time = pose.time_stamp;
+      ++pose_info.pose_count;
+      AINFO << "update start time: " << std::to_string(pose_info.start_time)
+            << ",pose count: " << pose_info.pose_count;
+    }
+  }
+
+  void ClearPoseInfo(std::shared_ptr<BadOrGoodPoseInfo> sp_pose_info) {
+    BadOrGoodPoseInfo& pose_info = *sp_pose_info;
```
Comment (anyong298) [inline modules/map/tools/map_datachecker/server/alignment.h]: Before using a shared_pointer, it's better to check whether the pointer is nullptr.

File: modules/map/tools/map_datachecker/server/channel_verify_agent.cc [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    case CmdType::STOP:
+      AINFO << "ChannelVerifyAgent stop";
+      StopCheck(request, response);
+      break;
+    default:
+      response->set_code(ErrorCode::ERROR_REQUEST);
+      AERROR << "command error";
+  }
+  AINFO << "ChannelVerifyAgent progress: " << response->DebugString();
+  return grpc::Status::OK;
```
Comment (anyong298) [inline modules/map/tools/map_datachecker/server/channel_verify_agent.cc]: The returned value is always OK. Is this desired?

Comment (ooooooops) [reply modules/map/tools/map_datachecker/server/channel_verify_agent.cc]: 是的，grpc服务要求返回值，其他返回值一般用不到

**Pull request merged**